style/style.css

```diff
@@ -0,0 +1,55 @@
+/*
+    ==============================================
+                ****** RESET ******
+    ==============================================
+*/
+body {
+  margin: 0;
+}
+
+:root {
+  --color-red: #f42;
+  --bg-color-red: #f42;
+}
+
+*,
+*::before,
+*::after {
```

Comment:
You do not use any ::before/ ::after selectors in your project. You can remove them.

Reply:
I think this **`change requirement`** would have been much better if a small explanation on **`why`** the change was important would have been shared as well.